src/lightning/data/processing/image.py

```diff
@@ -0,0 +1,47 @@
+import io
+from typing import Optional, Tuple
+
+from lightning_utilities.core.imports import RequirementCache
+
+_HTTPX_AVAILABLE = RequirementCache("httpx")
+
+# Credit to the https://github.com/rom1504/pytorch Github repo
```

Comment:
link 404's
Also we need to mention the license here

Reply:
done.